README.md

```diff
@@ -1,91 +1,159 @@
-  that will allow to interprete them and provide universal processing to the
-  possible extent if the remote sides trust each other and formalization of
-  data doesn't make sense.
-
-## Getting Started
-
-JSTP works in Node.js and web browsers. To install it, simply run:
+<div align="center">
+  <a href="https://github.com/metarhia/JSTP">
+    <img
+      src="https://cdn.rawgit.com/metarhia/Metarhia/master/Logos/metarhia-logo.svg"
+      alt="Metarhia Logo"
+      width="300"
+    />
+  </a>
+  <br />
+  <br />
+  <a href="https://travis-ci.org/metarhia/JSTP">
+    <img
+      src="https://travis-ci.org/metarhia/JSTP.svg?branch=master"
+      alt="Travis CI"
+    />
+  </a>
+  <a href="https://www.bithound.io/github/metarhia/JSTP/master/dependencies/npm">
+    <img
+      src="https://www.bithound.io/github/metarhia/JSTP/badges/dependencies.svg"
+      alt="bitHound Dependencies"
+    />
+  </a>
+  <a href="https://www.bithound.io/github/metarhia/JSTP">
+    <img
+      src="https://www.bithound.io/github/metarhia/JSTP/badges/score.svg"
+      alt="bitHound Score"
+    />
+  </a>
+  <a href="https://badge.fury.io/js/metarhia-jstp">
+    <img
+      src="https://badge.fury.io/js/metarhia-jstp.svg"
+      alt="NPM Version"
+    />
+  </a>
+  <a href="https://www.npmjs.com/package/metarhia-jstp">
+    <img
+      src="https://img.shields.io/npm/dm/metarhia-jstp.svg"
+      alt="NPM Downloads/Month"
+    />
+  </a>
+  <a href="https://www.npmjs.com/package/metarhia-jstp">
+    <img
+      src="https://img.shields.io/npm/dt/metarhia-jstp.svg"
+      alt="NPM Downloads"
+    />
+  </a>
+  <h1>JSTP / JavaScript Transfer Protocol</h1>
+</div>
+
+JSTP is an RPC protocol and framework which provides two-way asynchronous data
+transfer with support of plentiful parallel non-blocking interactions that is so
+transparent that an app may not even distinguish between local async functions
+and remote procedures.
+
+And, as a nice bonus, there's a blazing fast [JSON5](https://github.com/json5)
+implementation bundled in!
+
+**This project is bound by a [Code of Conduct](CODE_OF_CONDUCT.md).**
+
+## Installation
+
+JSTP works in Node.js and web browsers:
 
 ```sh
 $ npm install --save-dev metarhia-jstp
 ```
 
-Or, alternatively, if you are developing a client-side application for browser
-environment and don't use a module bundler like Webpack,
+Or, alternatively, there are
 [jstp.min.js](https://metarhia.github.io/JSTP/dist/jstp.min.js) and
-[jstp.min.js.map](https://metarhia.github.io/JSTP/dist/jstp.min.js.map) are
-also available.
+[jstp.min.js.map](https://metarhia.github.io/JSTP/dist/jstp.min.js.map)
+available for those browser-based applications that aren't built using a module
+bundler like [webpack](https://webpack.js.org/).
+
+We also have official client-side implementations for
+[Swift](https://github.com/JSTPMobile/iOS) and
+[Java](https://github.com/JSTPMobile/Java) that work effortlessly on iOS and
```

Comment:
Maybe it's better to put 'that work...' on a same line as 'Android', it may be easier to edit later + better diff in case we need to.